### DISCUSSION TITLE
More hallucinations on longer audio files

### DISCUSSION THREAD

**Discussion by bfavero** [SYSTRAN/faster-whisper#459] [Q&A]
I've been using Faster Whisper successfully for a while now, but I've noticed that longer audio files tend to have more hallucinations. I've run a few tests using the same files trimmed to different sizes. Consistently, the same snippets are more accurately transcribed when the file is shorter and there are more hallucinations when it's super long (> 3h)

Does anyone have a similar experience? Is there a reasonable explanation for why this may be happening?

**Comment by Purfview** [comment]
Try `prompt_reset_on_temperature=0`, it will reset prompt when model starts "fantasizing".

**Reply by bfavero** [reply]
Thanks! I tried that, still getting the same results, though

**Reply by Purfview** [reply]
Then maybe it's coincidence that you got better accuracy with trims.
With trims you actually are resetting prompt at random places, if you think that prompt is a bad thing for your audios then you can disable it: `condition_on_previous_text=False`.

Btw, what settings do you use for transcription?
